pkg/webhooks/webhookpatch.go

```diff
@@ -172,18 +177,21 @@ func (w *WebhookCertPatcher) patchMutatingWebhookConfig(
 	webhookConfigName string) error {
 	config, err := client.Get(context.TODO(), webhookConfigName, metav1.GetOptions{})
 	if err != nil {
+		reportWebhookPatchFailure(webhookConfigName, reasonWebhookConfigNotFound)
```

Comment:
this code is only for mutating webhook - should we apply the same metrics to validating webhook as well?